requirements-client.txt

```diff
@@ -0,0 +1,4 @@
+aiida-core==1.2.1
+ase==3.19.1
+numpy==1.18.4
+pymatgen==2020.6.8
```

Comment:
I specifically wanted to avoid this, hence the different extras, but is this only for the CI or?

Reply:
This is for the CI, so dependabot can basically bump and test new versions as they come out.

Reply:
I thought it could do that if the requirements were listed in `setup.py` as well, even for extras? At least I think we had it do this for the `django` dependency, which is an extra.

Reply:
Nope.  It's been using the requirements file in `.github` folder.
Note the files changed in this PR: https://github.com/Materials-Consortia/optimade-python-tools/pull/291